CMSIS/NN/Tests/UnitTest/TestCases/TestData/maxpooling_7/config_data.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2010-2021 Arm Limited or its affiliates. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Generated by generate_test_data.py using TFL version 2.4.1 as reference.
+#pragma once
+#define MAXPOOLING_7_OUT_CH 1
+#define MAXPOOLING_7_IN_CH 1
+#define MAXPOOLING_7_INPUT_W 4
+#define MAXPOOLING_7_INPUT_H 2
+#define MAXPOOLING_7_DST_SIZE 2
+#define MAXPOOLING_7_INPUT_SIZE 8
+#define MAXPOOLING_7_INPUT_OFFSET -128
+#define MAXPOOLING_7_OUTPUT_OFFSET 0
+#define MAXPOOLING_7_OUT_ACTIVATION_MIN 0
```

Comment:
Looking at the activation min/max all other test cases have the int8 min/max values.. And its good that this one has 0 as min.. but, would be good to have non-127 as max as well..could you update that either in this TC or in another one..?
 Also, could you make the output offset non-zero? all have zero right now.

Reply:
Ok, removing offsets for the pooling tests as agreed.